### PR TITLE
Adds Dildos!

### PR DESCRIPTION
**Pull request by tigercat2000**
Whatwhat! We got all kinds of dildos up in here!

We got Sea Dragon dildos (ribbed for your pleasure), Vulvakin dildos
(spiked for your masochism), Equine dildos!
We even have metal dildos, for the IPC crowd!

mmmmmmmmmmmmmm

### CODE REVIEW

**Comment by Gummin** [comment]
lewd

**Comment by Spacemanspark** [comment]
Why would we use this.
In any scenario.

File: code/modules/tiger/dildo.dm [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+	icon_state = "seadragon"
+
+	w_class = 2
+
+
+/obj/item/weapon/dildo/canine
+	name = "\improper Vulvakin Dildo"
+	desc = "A Canine Dildo, designed to look very close to those of the Vulvakin species."
+
```
Comment (Iamgoofball) [inline code/modules/tiger/dildo.dm]: >all this fucking whitespace

this is some kind of joke isnt it

**Comment by Iamgoofball** [comment]
# #3333

**Comment by KasparoVy** [comment]
dude nice quads
333333333333333333333333333333333333333333333333333333333333333333333333333333333

top kek descriptions

**Comment by Spacemanspark** [comment]
I was wondering if this was a joke.

**Comment by KasparoVy** [comment]
it's absolutely srs

**Comment by tigercat2000** [comment]
@Spacemanspark 
The real question you should be asking is, why **wouldn't you** use this in
# EVERY SITUATION?

Here at the Tiger Dildo Emporium, we have a large selection of dildos! Tentacle dildos, furry dildos, metal dildos, and many many more! Buy yours today for only 5312.5 GNB (Generic NanoTrasen Bucks are a trademark of NanoTrasen Inc. and are not in any way affiliated with the Tiger Dildo Emporium&trade;.

**Comment by Spacemanspark** [comment]
I can light them all on fire instead.

File: code/modules/tiger/dildo.dm (hunk 1 above)
Comment (tigercat2000) [inline code/modules/tiger/dildo.dm]: I assure you not, good sir @iamgoofball! The white space is completely intentional, to enhance the length and girth of the Tiger Dildo Emporium&trade;'s many dildos!

**Comment by KasparoVy** [comment]
dude that's mint

idk what kind of heat fetish you're into but i bet they've got something for it

**Comment by Gummin** [comment]
Please add Valve!

**Comment by KasparoVy** [comment]
not rekt [ ] | rekt [v/]

**Comment by tigercat2000** [comment]
01000101 01010010 01010010 00111010 00100000 01010011 01010100 01000001 01000011 01001011 00100000 01001111 01010110 01000101 01010010 01000110 01001100 01001111 01010111 00111010 00100000 01011010 01011010 01000010 01000001 01000010 01000001 00110011 01011010 01011010 01010011 01000101 01000011 01010010 01000101 01010100 01010011 01011010 01011010

**Comment by KasparoVy** [comment]
tfw the stack overflows

**Comment by tigercat2000** [comment]
Oh dear! I do believe my keyboard is currently steaming!

**Comment by tigercat2000** [comment]
first to find all three secrets gets a free dildo

**Comment by Gummin** [comment]
i dont get it

**Comment by KasparoVy** [comment]
branch name
ERR: STACK OVERFLOW: ZZBABA3ZZSECRETSZZ
Larkens station

laughed like an idiot when i found the donglord section "The [src] has chosen you to be it's new master!"

**Comment by tigercat2000** [comment]
@KasparoVy good guesses but those aren't quite what I mean

the secrets are all in the file diff

**Comment by KasparoVy** [comment]
god damn the stats on the horse
you could proper kill someone with that

**Comment by tigercat2000** [comment]
you still get a free dildo though
:pencil2: :+1: :poodle: = :astonished:

**Comment by Fox-McCloud** [comment]
Joke has run its course methinks.

**Comment by Iamgoofball** [comment]
Fuck off Fox, this is a #3333 pr, reopen

File: code/modules/tiger/dildo.dm [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+	name = "purple dong"
+	desc = "Reminiscent of your mother's dildo."
+
+	icon_state = "purple-dong"
+	item_state = "purple-dong"
+
+	slot_flags = SLOT_BELT
+
+/obj/item/weapon/dildo/floppydick
+	name = "floppy dick"
```
Comment (Iamgoofball) [inline code/modules/tiger/dildo.dm]: >doesnt have a 50% chance to have a diskette sprite

**Comment by tigercat2000** [comment]
huh, my reopen button is gone. good job github, i think.

Also, secrets list of arbitrary secrets:
 - Purple dildo description is a reference to [Who's Your Daddy](http://store.steampowered.com/app/427730/)
 - Vulvakin dildo
 - Females get a special text when examining floppy dick dildo, `It's floppy. Very familiar!`
 - `Once you go black, you never go back!`

**Comment by Iamgoofball** [comment]
Congrats on losing your maintainer status

**Comment by FalseIncarnate** [comment]
Tiger wasn't a maintainer goof.

Also, damn it tiger, you forgot to confirm illuminati with this PR.

**Comment by tigercat2000** [comment]
THAT'S WHAT THEY WANT YOU TO THINK (whoops)

also i think it's from switching the repo to an organization

**Comment by Plotron** [comment]
Damn, I missed some jolly good fun.

**Comment by Tastyfish** [comment]
Christ, why is this PR actually functional and sprited.

**Comment by NTSAM** [comment]
holy fuck

**Comment by Spacemanspark** [comment]
@Tastyfish
Because these are actually a thing on Hippie Station.

**Comment by Tastyfish** [comment]
I figured it was from some other server.

**Comment by tigercat2000** [comment]
@Spacemanspark @Tastyfish it's from hippie and a random ERP server

**Comment by Spacemanspark** [comment]
Citadel?

**Comment by tigercat2000** [comment]
@spacemanspark ew no

**Comment by Spacemanspark** [comment]
:^)

**Comment by Rumiluntti** [comment]
Air left rhytmically from my nasal cavity with abnormal velocity.
Good shit

**Pull request closed without merging**